### PR TITLE
bump-packages: better handle action inputs

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
Store the formulae and casks in arrays to eliminate code injection through those inputs.

Also, update the descriptions to note that inputs should be space-separated.

### CODE REVIEW

File: bump-packages/action.yml [hunk 1]
```diff
@@ -9,11 +9,11 @@ inputs:
     description: GitHub token (not the default one)
     required: true
   formulae:
-    description: Formula names, one per line
+    description: Formula names, space-separated
     required: false
```
Comment (Bo98) [inline bump-packages/action.yml:13]: Is this a breaking change or was the description wrong before?

Comment (ZhongRuoyu) [reply bump-packages/action.yml:13]: I believe it was wrong (or rather a misunderstanding of how YAML `>` syntax works).

If it really is one per line then this probably wouldn't work:

https://github.com/Homebrew/actions/blob/250a86a75e69d18d941481f9c02b684363233daf/bump-packages/action.yml#L28

Comment (Bo98) [reply bump-packages/action.yml:13]: Cool, make sense!

**Review by woodruffw — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks @ZhongRuoyu!

**Comment by Bo98** [comment]
Seeing some bad bumps after this: https://github.com/Homebrew/homebrew-core/pull/177286

**Comment by ZhongRuoyu** [comment]
Whoops, should be fixed by https://github.com/Homebrew/actions/pull/565!

**Comment by ZhongRuoyu** [comment]
Now working as expected: https://github.com/Homebrew/homebrew-core/actions/runs/9927784994/job/27423178212

**Pull request merged**